src/pages/home/Home.js

```diff
@@ -0,0 +1,85 @@
+import React, {useEffect, useState} from "react";
+import "./Home.css";
+import Header from "../../components/header/Header";
+import logo from "../../assets/reddit_logo.png";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import titleLength from "../../helperfuncties/titleLength";
+import numberDots from "../../helperfuncties/numberDots";
+import Footer from "../../components/Footer";
+
+function Home() {
+    const [redditData, setRedditData] = useState("");
```

Comment:
Als `redditData` uiteindelijk een array is, dan zou ik de default waarde óók een array maken
```suggestion
    const [redditData, setRedditData] = useState([]);
```